src/NzbDrone.Core/Extras/Subtitles/SubtitleService.cs

```diff
@@ -87,7 +87,8 @@ public override ExtraFile Import(Series series, EpisodeFile episodeFile, string
             if (SubtitleFileExtensions.Extensions.Contains(Path.GetExtension(path)))
             {
                 var language = LanguageParser.ParseSubtitleLanguage(path);
-                var suffix = GetSuffix(language, 1, false);
+                var forced = LanguageParser.ParseForcedFlag(path);
```

Comment:
Parsing `forced` here makes sense, it's specific to subtitles, not general language parsing.

Tests for it should also be added.